### PR TITLE
docs(README): Created Installation section

### PR DESCRIPTION
**Pull request by Dosx001**
# Description
I just updated the README.

I also packaged languagetool-rust for Arch Linux.
https://aur.archlinux.org/packages/languagetool-rust

I added a section on how to install languagetool-rust on Arch Linux using the AUR.

If you release an update for languagetool-rust, you can ping me, and I'll update the package.

### CODE REVIEW

**Comment by jeertmans** [comment]
Thanks @Dosx001!
Before merging this, I have a small question / comment.

What is the process required to deploy a Arch package?

if this is not complicated, I suggest to automate it. Either you create a GitHub action job that uses some token to automatically publish every time I publish.

You could also use dependabot on another repo you own, watch for updates and be alerted every time I update. Because I cannot promise to ping you every time I upgrade :’-)

**Comment by Dosx001** [comment]
@jeertmans it's not that complicated but as of right now I am the maintainer for languagetool-rust on the AUR. I am not sure about this but I believe you would need my SSH Public Key to push changes to the repo. I will have to look into this. I wouldn't mind making this a GitHub action.

**Comment by Dosx001** [comment]
Looks like there are ways to send notifications using GitHub Actions.

On Discord
https://www.youtube.com/watch?v=-KDQqWNK3Tw
And Slack
https://www.youtube.com/watch?v=RKfsnp_AN-4

This might be an easier approach. There doesn't seem to be a way to have a forked repo that runs an action when upstream does a release.

**Comment by Dosx001** [comment]
WAIIIITTTT!!! LLLOLOLOLOLOLOLO!!! I just could watch the repo and turn on notifications on release.  lol

File: README.md
Comment (jeertmans) [inline README.md:38]:
```suggestion
```

I don't think LTRs needs to tell that. This should be performed only once, and most likely when installing Cargo.

File: README.md
Comment (jeertmans) [inline README.md:42]:
```suggestion
If you are on Arch Linux, you call also install with the package manager:
```

**Review by jeertmans — CHANGES_REQUESTED** [review]
Just a few suggested changes :-)

Also, please write something in the CHANGELOG.md, as suggested by the CI :-)

**Comment by jeertmans** [comment]
Merged @Dosx001 ! Thanks :-)

**Pull request merged**
